com.adobe.Flash-Player-Projector.appdata.xml

```diff
@@ -4,6 +4,9 @@
   <id>com.adobe.Flash-Player-Projector</id>
   <name>Adobe Flash Player</name>
   <summary>Player for content created using Adobe Flash</summary>
+  <developer id="com.adobe">
```

Comment:
You should remove that bit, it's already there 2 lines below (added in https://github.com/flathub/com.adobe.Flash-Player-Projector/pull/71)